tests/unit/replication/UpdateReplicationStatus.js

```diff
@@ -31,6 +33,9 @@ function checkReplicationInfo(site, status, updatedSourceEntry) {
         updatedSourceEntry.getReplicationSiteStatus(site), status);
 }
 
+const metricsHandlers = ReplicationStatusProcessor.loadMetricHandlers(
```

Comment:
```suggestion
const metricHandlers = ReplicationStatusProcessor.loadMetricHandlers(
```
Kinda nit picky but I think "metric handlers" makes more sense then "metrics handlers"

Reply:
This seems to have been a typo we made somewhere and I see both being used so maybe we can just swap them all to the same "metric handlers" now.

Reply:
I believe the "metrics" word is actually singular in this domain, and we should always speak of "metrics" even if there is a single one... so metricsHandler seems fine with me.